application/asset/sass/_screen.scss

```diff
@@ -4111,6 +4112,11 @@ body.minimal {
             margin: 0 0 0 $spacing-medium;
             line-height: 20px;
         }
+        
+        .site-thumbnail-image {
+            max-width: 25%;
+            float: left;
+        }
```

Comment:
Where we're floating the thumbnails, I think a right margin (maybe 1rem?) on the thumbnails would be a good default.